frontend/src/components/organisms/ManageUserProfileNew.tsx

```diff
@@ -143,7 +153,7 @@ const ManageUserProfileNew = () => {
     email,
     joinDate,
     userId,
-    hours,
+    hours: fakeHours,
```

Comment:
What's this about?

Reply:
I can't delete the `hours` column from the `UserProfile` column, so we still grab the hours from the profile but we can't use it. I use `hours` for the actual hours and `fakeHours` for the hours grabbed from profile.